Playbooks/AS_Alert_Spiderfoot_Scan/azuredeploy.json

```diff
@@ -15,8 +15,8 @@
             "type": "string"
         },
         "SpiderfootAPIKey": {
-            "defaultValue": "Enter your Spiderfoot API key here.",
```

Comment:
I don't have access to revert this commit, can you remove this file from this PR?

Reply:
Unfortunately, I am only working from the web ui and there doesn't appear to be a to do this. I have created a new PR with all of the changes from this one which can be found here: https://github.com/Azure/Azure-Sentinel/pull/4687